.upstream-tests/test/std/iterators/iterator.container/ssize.pass.cpp

```diff
@@ -142,7 +143,9 @@ int main(int, char**)
     static_assert( cuda::std::is_signed_v<                      decltype(cuda::std::ssize(sc))>, "");
     static_assert( cuda::std::numeric_limits<                   decltype(cuda::std::ssize(sc))>::max()  > 60000, "");
     static_assert( cuda::std::numeric_limits<cuda::std::make_signed_t<decltype(cuda::std:: size(sc))>>::max() < 60000, "");
+#ifdef __CUDA_ARCH__
```

Comment:
Why?

Reply:
This is again some very strange compiler bug that leads to wraparound on device

I should have added comments to those 🤦